service/src/main/kotlin/app/cash/backfila/service/SlackHelper.kt

```diff
@@ -13,7 +12,7 @@ class SlackHelper @Inject constructor(
   @BackfilaDb private val transacter: Transacter,
   private val slackClient: SlackClient,
   private val backfilaConfig: BackfilaConfig,
-  private val env: Env
+  private val deployment: wisp.deployment.Deployment
```

Comment:
why not import this?

Reply:
Can clean it up, it might have been during the transition phase when both wisp. and misk. existed.